silkworm/silkrpc/core/evm_trace.cpp

```diff
@@ -1566,7 +1566,17 @@ Task<void> TraceCallExecutor::trace_filter(const TraceFilter& trace_filter, cons
     SILK_INFO << "TraceCallExecutor::trace_filter: filter " << trace_filter;
 
     const auto from_block_with_hash = co_await core::read_block_by_number_or_hash(block_cache_, storage, database_reader_, trace_filter.from_block);
+    if (!from_block_with_hash) {
+        const Error error{-32000, "invalid parameters: fromBlock not found"};
+        stream->write_field("error", error);
+        co_return;
+    }
     const auto to_block_with_hash = co_await core::read_block_by_number_or_hash(block_cache_, storage, database_reader_, trace_filter.to_block);
+    if (!to_block_with_hash) {
+        const Error error{-32000, "invalid parameters: fromBlock not found"};
+        stream->write_field("error", error);
+        co_return;
+    }
```

Comment:
Another special case: streaming API. It seems OK here.

Reply:
yes